spec/support/feature_helpers.rb

```diff
@@ -1,7 +1,9 @@
 module FeatureHelpers
   def login_new_user(options = {})
     visit "/"
-    click_link "Sign in"
+    within(:css, ".super-button") do
```

Comment:
No problem! It was a good way to get familiar with the code base! I do practice TDD, so I'll certainly use it if I add any new features. 

Good catch. I'm not sure why I put that in there 😕